sdks/python/apache_beam/transforms/enrichment.py

```diff
@@ -54,6 +56,12 @@ def cross_join(left: Dict[str, Any], right: Dict[str, Any]) -> beam.Row:
     if k not in left:
       # Don't override the values in left.
       left[k] = v
+    elif left[k] != v:
+      _LOGGER.warning(
+          '%s exists in the input row as well the row fetched '
+          'from API but have different values. Using the input '
+          'value, you can override this behavior by passing a '
+          'custom `join_fn`.' % k)
```

Comment:
Lets log the values as well. Also, nit: `Using the input value, you can override this behavior ...` makes it sound like using an input you can override, maybe just split these into separate sentences.

Reply:
sounds good!